pkg/grafana/client/client.go

```diff
@@ -29,7 +29,7 @@ func GenerateGrafanaClient(ctx context.Context, client client.Client, logger log
 		return nil, fmt.Errorf("failed to build tls config: %w", err)
 	}
 
-	grafanaUrl, err := url.Parse(grafanaURL)
+	grafanaUrl, err := url.Parse(fmt.Sprintf(grafanaURL, namespace))
```

Comment:
```suggestion
	grafanaUrl, err := url.Parse(fmt.Sprintf(grafanaURL, grafanaNamespace))
```
my bad :D

Reply:
Don't worry I noticed and it's already fixed :)